### PR TITLE
Alyson's Individual Assignment

### PR DESCRIPTION
**Pull request by akr5030**
This is my individual assignment. I used categorization for the user interface pattern. I created a UI for the food view. I used abstract factory for the object-oriented design pattern. There is a meal interface and a meal factory.

### CODE REVIEW

**Review by redjen — CHANGES_REQUESTED** [review]
It looks like your Account-related changes are in here, which is going to cause issues with merging. Were you working on them in this change? If the version that you want for Accounts is in this PR then we could use it for both. Otherwise, I think you can undo that with:

git checkout src/accounts
git checkout src/dao
git rebase origin/master (to get the latest changes from the repository since Walt's login changes might get merged in first)
git status (then commit if needed)
git push

Checkout doesn't seem like the way to reset a file, but it is. I usually copy the file somewhere outside the project (like Desktop or whatever) just in case it has changes in it that I need. 

Usually when this happens to me it's because I did git checkout -b newBranchName while in one of my local branches instead of checking out master first.

File: src/foodmood/MealFactory.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package foodmood;
+
```
Comment (redjen) [inline src/foodmood/MealFactory.java:2]: This looks fine, but I don't see the Meal interface.

**Comment by akr5030** [comment]
The meal interface is there now. I'll work on fixing the issues with the Account-related changes.

**Comment by redjen** [comment]
If it's easier you could copy the meal files, git checkout master, make a new branch, then copy them into the new branch and use that. I usually do that when I run into this situation (which I do on the regular). You'd need a new PR, though. 

On October 22, 2017 1:15:09 PM PDT, akr5030 <notifications@github.com> wrote:
>The meal interface is there now. I'll work on fixing the issues with
>the Account-related changes.
>
>-- 
>You are receiving this because your review was requested.
>Reply to this email directly or view it on GitHub:
>https://github.com/redjen/FoodMood/pull/24#issuecomment-338505697

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Pull request closed without merging**